src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -52,7 +52,7 @@ const MY_TTL_SEC: u64 = 3600; // TODO: should be a long value, say 1 hour
 const MY_GENERATION: u64 = 1; // TODO: find a proper way to set generation
 #[allow(dead_code)]
```

Comment:
By the way, we can remove this `dead_code` mark now, as the `TXN_RETRY_LIMIT` is commonly used in any ABI versions.